Gems/Atom/RPI/Code/Source/RPI.Reflect/Image/StreamingImageAsset.cpp

```diff
@@ -124,5 +237,119 @@ namespace AZ
 
             return mipChainAsset->GetSubImageData(mip - mipChain.m_mipOffset, slice);
         }
+
+        template<typename T>
+        T StreamingImageAsset::GetSubImagePixelValueInternal(uint32_t x, uint32_t y, uint32_t componentIndex, uint32_t mip, uint32_t slice)
+        {
+            AZStd::vector<T> values;
+            values.resize(1);
```

Comment:
I'd use an AZStd::array here - no need to use heap memory.

Reply:
Will do!